bin/radical-pilot-create-static-ve

```diff
@@ -111,11 +111,27 @@ fi
 mkdir -p "$prefix"
 cd $prefix
 
-echo -n "install virtualenv "
+# for conda environments, install virtualenv in the env
+if ! test -z "$CONDA_DEFAULT_ENV"
+then
+    # conda developers did not think this through... :-(
+    CONDA="$(which conda 2>/dev/null | head -n 1)"
+    test -z "$CONDA" && CONDA="$_CONDA_EXE"
+    test -z "$CONDA" && CONDA="$CONDA_EXE"
+    if test -z "$CONDA"
+    then
+        echo "conda is not functional"
+        exit 1
+    fi
+    echo -n "install conda virtualenv "
+    "$CONDA" install -y virtualenv 2>&1 | progress || exit 1
```

Comment:
This is where the virtualenv is created, right? It should be `conda create` instead of `install`

Reply:
Ah, no - this is where `virtualenv` is installed in the conda env, so that the remainder of the script can then use `virtualenv` to create the pilot VE.  Yeah, one can also use conda to actually *create* the ve - but I wanted to keep the changes minimal, and not start testing how the pilot likes a such created VE.  With the current fix, the VE created by the script remains exactly as before.

Reply:
okay!